seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java

```diff
@@ -85,25 +90,29 @@ private Object get(String dataId, Object defaultValue) throws IllegalAccessExcep
         return result;
     }
 
-    private Object get(String dataId) throws IllegalAccessException, InstantiationException {
+    private Object get(String dataId) throws IllegalAccessException {
         String propertyPrefix = getPropertyPrefix(dataId);
         String propertySuffix = getPropertySuffix(dataId);
-        ApplicationContext applicationContext = (ApplicationContext) ObjectHolder.INSTANCE.getObject(OBJECT_KEY_SPRING_APPLICATION_CONTEXT);
         Class<?> propertyClass = PROPERTY_BEAN_MAP.get(propertyPrefix);
         Object valueObject = null;
         if (propertyClass != null) {
             try {
-                Object propertyBean = applicationContext.getBean(propertyClass);
-                valueObject = getFieldValue(propertyBean, propertySuffix, dataId);
+                valueObject = getFieldValue(
+                    Objects.requireNonNull(PROPERTY_BEAN_INSTANCE_MAP.computeIfAbsent(propertyPrefix, k -> {
+                        try {
+                            return propertyClass.newInstance();
```

Comment:
The lack of bean injection is a problem for the framework. I don't think we should use the default instance.